public/index.css

```diff
@@ -458,7 +457,43 @@ img {
 .summer-bg {
   background-image: url("./assets/images/bg_v01.webp");
 }
-
+#levelEnd{
+  display: none;
+  height: 100vh;
+  width: 100%;
+  position: fixed;
+  z-index: 7;
+  background-image: url(./assets/images/WIN_screen_bg.webp);
```

Comment:
Question, our level end scene has also seasonal backgrounds. Do we also handle that seasonal changes in here or somewhere?

Reply:
As the level end background is a static background so we can keep it separately in css, but all the seasonal background is already handled from background component and that will be visible through the gameplay background which is already rendered so no need to handle it separately.